src/main/java/frc/robot/subsystems/Drivetrain.java

```diff
@@ -243,6 +243,11 @@ public void resetEncoders() {
     m_rearRight.resetEncoders();
   }
 
+  /**
+   * Sets the idle states of the sparkMAX motors.
+   *
+   * @param mode the mode to set the states to (0 is coast, 1 is brake)
+   */
   public void setIdleStates(int mode) {
     m_frontLeft.setIdle(mode);
```

Comment:
Variables: No under_scores (except constants).

This whole class uses underscores for variable names, and should be fixed.
'm_frontLeft' should be 'mFrontLeft', 'frontLeftSwerveModule', or something like that.

Reply:
Our code is a fork of REV Robotics swerve code, and both REV and WPILib use Hungarian notation for their variables (Ex m_ to mark it is a member variable, k to mark it as a constant. If we are getting rid of Hungarian notation we need to remove the m_ and k prefix from a large amount variables in our code.

Reply:
Ok, I'll look into it, but still in java I believe we need to omit the under_score, if need fix, will be new PR.